starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,62 @@
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.image-container {
+  background-image: url(../images/apple-pie.jpg);
+  background-size: cover;
+  position:0.0;
+  margin:0;
+  font-size:4em; 
+  height: 80vh;
+  line-height: 80vh;
+}
+
+h1{
+  margin: 0 auto;  
+  text-align: center;
+  color:white;
+  font-weight: 100;
+}
+
+.container{
+ padding: 35px 0; 
+  margin: 0 auto;
+}
+
+.footer-img{
+  width: 35%;
+  margin: 0 30% 0 30%;
+  text-align: center;
+  padding: 35px 0 0 0;
+}
+
+hr {
+  border: solid 1.2px gray;
+  background-color: gray;
+}
+
+section {
+  width:70%;
+  margin: 0 auto;
+}
+
+#intro p {
+  padding:0px 0 0 0;
+  font-size: 1.5em;
+}
+
+.text{
+ font-size: 1em;
+}
+
+ul {
+  list-style-type: none;
+}
+
+ul li:before {
```

Comment:
Smart stuff! This is a really good way to get the bullet points to appear as dashes.